crons/image-monitor.sh

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env bash
+#
+# Find if a new tag has been published to dockerhub after the given tag.
+# Only works for new tags, not tag updates.
+# For tag updates, use Diun
+#
+set -eufo pipefail
+
+SCRIPT_DIR="$(cd "$(dirname "$0")" && pwd)"
+
+ERROR_CODE_INVALID_USAGE=1
+ERROR_CODE_INVALID_INPUT=2
+
+SLACK_URL="https://slackbot.internal.aleemhaji.com/message"
+SLACK_CHANNEL="CKE1AKEAV"
+
+ignore_tags="latest edge nightly beta preview unstable dev stable testing"
+global_ignore_tags_selector=""
+for rtag in $ignore_tags; do
+    global_ignore_tags_selector="$global_ignore_tags_selector | select(test(\"^$rtag$\") | not)"
+done
+
+all_ignore_tags="$(jq -r '. | tostring' "$SCRIPT_DIR/image-monitor-ignore.json")"
+
+slack() {
```

Comment:
Move this to use the shell monitor? Might be nicer to have one complete message than tons of messages every cron interval, plus, only messages on content change, which _should_ be less annoying.